exp-player/addon/components/exp-frame-base/component.js

```diff
@@ -139,7 +139,7 @@ export default Ember.Component.extend({
             this.send('setTimeEvent', 'nextFrame');
             this.send('save');
             this.sendAction('next');
-            window.scrollTo(0, 0);
+            //window.scrollTo(0, 0);
```

Comment:
That sounds like a reset bug worth revisiting later, but reverting this change for now would help.

We can create a followup ticket for the issue you report, which may be the preferred way to target a fix so we don't forget to coordinate across projects after the holiday weekend.